src/scoping-shim.js

```diff
@@ -512,7 +512,13 @@ export default class ScopingShim {
   // any necessary ShadyCSS static and property based scoping selectors
   setElementClass(element, classString) {
     let root = StyleUtil.wrap(element).getRootNode();
-    let classes = classString ? classString.split(/\s/) : [];
+    let classes;
+    if (classString) {
```

Comment:
This conditional still doesn't match native behavior for falsey values - which is to ToString the value no matter what. We should see `false`→`"false"`, `undefined`→`"undefined"`, etc.

I'd probably write this as:

```js
classes = String(classString).split(/\s/);
```

Reply:
I had this originally, but Peter mentioned that there was a performance hit on calling String() on a string :/

Reply:
Justin is the one that I got the wisdom of not always calling String() from, so his opinion should override mine here

Reply:
There is a small cost, but I was more concerned with this in a code path in lit-html that is easily called 10x or more than this one. Even then it was slightly debatable to do a manual optimization rather than push browsers to do better in `String()`.

We could do the same here though:

```js
const classes = (typeof classString === 'string' ? classString : String(classString)).split(/\s/);
```